### PR TITLE
test: adds attributes for testing

### PR DESCRIPTION
**Pull request by vilkg**
Attributes that help with test case implemented in [ this PR. ](https://github.com/dhis2/e2e-tests/pull/48). The test case should cover this scenario https://jira.dhis2.org/browse/DHIS2-5287, that is part of QA teams regression testing activities. 

Tested locally.

### CODE REVIEW

**Comment by paschalidi** [comment]
Happy to review @vilkg. Wanna add a short description explaining the motivation behind the `data-test`s? That would give us some perspective :)

File: src/core_modules/capture-core/components/Pages/MainPage/MainPage.component.js [hunk 1]
```diff
@@ -38,6 +38,7 @@ class Index extends Component<Props> {
                             <TrackerProgramHandler>
                                 <div
                                     className={classes.listContainer}
+                                    data-test={'dhis2-capture-event-list-container'}
```
Comment (paschalidi) [inline src/core_modules/capture-core/components/Pages/MainPage/MainPage.component.js:41]:
```suggestion
                                    data-test="dhis2-capture-event-list-container"
```

**Comment by vilkg** [comment]
Hi @paschalidi, 

I fixed the suggestions, thanks for that!

File: src/core_modules/capture-core/components/LockedSelector/QuickSelector/OrgUnitSelector.component.js [hunk 2]
```diff
@@ -168,11 +168,12 @@ class OrgUnitSelector extends Component<Props, State> {
         }
 
         return (
-            <div>
+            <div data-test="dhis2-capture-org-unit-selector-container"t>
```
Comment (JoakimSM) [inline src/core_modules/capture-core/components/LockedSelector/QuickSelector/OrgUnitSelector.component.js:171]:
```suggestion
            <div data-test="dhis2-capture-org-unit-selector-container">
```

**Comment by vilkg** [comment]
Thanks @JoakimSM , fixed

**Pull request merged**